src/platform/qt/Window.cpp

```diff
@@ -1630,6 +1630,11 @@ void Window::setupMenu(QMenuBar* menubar) {
 	m_actions.addAction(tr("Game Pak sensors..."), "sensorWindow",  openNamedControllerTView<SensorView>(&m_sensorView, &m_inputController), "tools");
 
 	addGameAction(tr("&Cheats..."), "cheatsWindow", openControllerTView<CheatsView>(), "tools");
+	addGameAction(tr("&Disable cheats..."), "disableCheats", [this]() {
```

Comment:
```suggestion
	addGameAction(tr("Disable all cheats"), "disableCheats", [this]() {
```

The "..." at the end indicates that it opens a new window, which this one doesn't.